doc/source/sep/index.md

```diff
@@ -0,0 +1,55 @@
+Sardana Enhancement Proposals
+=============================
+
+This is the main index of the Sardana Enhancement Proposals (SEP).
+
+Each proposal should be in a separate file and be linked in the following table
+
+Proposals list
+--------------
+
+  Link        |  status   |        Title
+  ------------| --------- | ---------------------------------------------------------
+  [SEP0][]    | OBSOLETE     | Introducing Sardana Enhancement Proposal
+  [SEP1][]    | OBSOLETE     | Reorganization of code repos
+  [SEP2][]    | DRAFT     | Lima integration
+  [SEP3][]    | REJECTED (handled in [#297][]) | Adapt to [TEP3][] (Tango-independent taurus.core)
+  [SEP4][]    | ACCEPTED     | HKL integration
+  [SEP5][]    | ACCEPTED     | Implementation of tests infrastructure
+  [SEP6][]    | CANDIDATE     | Continuous Scan Implementation
+  [SEP7][]    | OBSOLETE     | Code contribution workflow
+  [SEP8][]    | CANDIDATE     | Remove from Taurus objects the direct Logger dependence
+  [SEP9][]    | ACCEPTED     | Compact Read+Write widgets in Taurus
+  [SEP10][]    | OBSOLETE     | Taurus separation
+  [SEP11][]    | ACCEPTED     | Direct load of .ui files
+  [SEP12][]    | CANDIDATE     | Use python Enum instead of taurus Enumeration
+  [SEP13][]    | REJECTED (moved to [TEP13][])     | Unified plugins support in Taurus & Sardana
+  [SEP14][]    | DRAFT     | MSENV taurus schema
+  [SEP15][]   | CANDIDATE     | Moving Sardana to Github
+
+
+
+
+[SEP0]: sep/SEP0.md
```

Comment:
I think these links will be broken. I recomend using links such as:
http://www.taurus-scada.org/tep/?TEP13.md